controller/array_action/array_mediator_ds8k.py

```diff
@@ -302,16 +303,16 @@ def delete_volume(self, volume_id):
         self._delete_volume(volume_id)
         logger.info("Finished deleting volume {}".format(volume_id))
 
-    def get_volume(self, name, volume_context=None):
-        logger.debug("Getting volume {} under context {}".format(name, volume_context))
-        if not volume_context:
+    def get_volume(self, name, pool_id=None):
+        logger.debug("Getting volume {} in pool {}".format(name, pool_id))
+        if not pool_id:
```

Comment:
refactor to `if pool_id is not None:` to distinguish between unspecified vs. empty pool parameter

Reply:
* if pool_id is None:

Reply:
done

Reply:
right